AirCasting/SessionViews/EditViewModel.swift

```diff
@@ -0,0 +1,57 @@
+// Created by Lunar on 15/12/2021.
+//
+
+import Foundation
+import SwiftUI
+import CoreData
+
+class EditSessionViewModel: ObservableObject {
+    
+    @Published var sessionName = ""
+    @Published var sessionTags = ""
+    private let measurementStreamStorage: MeasurementStreamStorage
+    private let sessionUpdateService: SessionUpdateService
+
+    init(measurementStreamStorage: MeasurementStreamStorage, sessionUpdateService: SessionUpdateService) {
+        self.measurementStreamStorage = measurementStreamStorage
+        self.sessionUpdateService = sessionUpdateService
+    }
+    
+    func saveChanges(for uuid: SessionUUID, completion: @escaping () -> Void) {
+        guard !sessionName.isEmpty else { return }
```

Comment:
Should we add an alert here, that the name can't be empty?

Reply:
I'm not sure, I didn't do it because we don't do that when creating a session. I'm wondering if it actually can be empty - it can be when creating a session. What do you think @anna1901  & @jankrzempek ?

Reply:
Couldn't this happen when a user edits name to an empty string?

Reply:
Yes, I'm not saying it's impossible. I'm saying that we're allowing this situation when you create a session. We don't have any alert and we're just creating a no-name session. 
So maybe this `guard let` is unnecessary in the first place because right now the session can have an empty name 🤷🏻‍♀️

Reply:
Alert added :)